engine/language-server/src/main/scala/org/enso/languageserver/search/SuggestionsHandler.scala

```diff
@@ -385,6 +385,17 @@ final class SuggestionsHandler(
           )
         }
         .pipeTo(sender())
+      if (state.shouldStartBackgroundProcessing) {
+        runtimeConnector ! Api.Request(Api.StartBackgroundProcessing())
```

Comment:
Does this mean that `SuggestionHandler` emits the `Api.StartBackgroundProcessing` message? If so, it is strange. I'd expect some more central authority to emit such message.